src/libs/Navigation/helpers/isReportOpenInSuperWideRHP.ts

```diff
@@ -0,0 +1,17 @@
+import type {NavigationState} from '@react-navigation/native';
+import {SUPER_WIDE_RIGHT_MODALS, WIDE_RIGHT_MODALS} from '@components/WideRHPContextProvider/WIDE_RIGHT_MODALS';
+import NAVIGATORS from '@src/NAVIGATORS';
+
+const isReportOpenInSuperWideRHP = (state: NavigationState | undefined): boolean => {
```

Comment:
Could you add a doc explaining what this means for folks, what the super wide rhp is etc?